gson/src/test/java/com/google/gson/internal/UnsafeAllocatorInstatiationTest.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright (C) 2016 Ugljesa Jovanovic
```

Comment:
Sorry, we won’t accept pull requests unless you assign copyright to either Google or Gson Authors. (We don’t want to have to get your permission later if we need to change the license of the whole project.)